src/Pipeline.cpp

```diff
@@ -555,6 +566,21 @@ Target Pipeline::get_compiled_jit_target() const {
 }
 
 void Pipeline::compile_jit(const Target &target_arg) {
+    std::string debug_serialization = get_env_variable("HL_DEBUG_SERIALIZATION");
+    Pipeline deserialized_pipe;
+    if (!debug_serialization.empty()) {
+        std::string filename = generate_function_name() + ".hlpipe";
+        debug(0) << "intercept compile_jit, file: " << filename << "\n";
```

Comment:
I assume that the `debug(0)` will be either removed or converted to `debug(1)` or 2 or 3, etc before landing?